avi/resource_avi_cloud.go

```diff
@@ -236,10 +243,104 @@ func ResourceAviCloudRead(d *schema.ResourceData, meta interface{}) error {
 	return err
 }
 
+// Verify cloudState is valid vcenter state to configure mgmt IP
+func isValidCloudState(cloudState string) bool {
```

Comment:
This is not intuitive. The other states are also valid states. May be call it isCloudNotReady